README.md

```diff
@@ -36,6 +36,7 @@ Awesome software projects sub-categorized by focus.
 <a name="simulation-and-modelling"></a>
 
 - ![Awesome](media/icon/awesome.png) [ESSM](https://github.com/environmentalscience/essm) – ![Python](media/icon/python.png) A package for traceable and reproducible mathematical derivations and analysis.
+- ![Awesome](media/icon/awesome.png) [Basic Model Interface (BMI)](https://github.com/csdms/bmi) – ![C](media/icon/c.png) ![C++](media/icon/cplusplus.png) ![Fortran](media/icon/fortran.png) ![Python](media/icon/python.png) A standardized set of functions for model-model and model-data coupling
 - ![Awesome](media/icon/awesome.png) [Landlab](https://github.com/landlab/landlab) – ![Python](media/icon/python.png) Landlab is a Python framework for building and running numerical models of Earth surface dynamics
 - ![Awesome](media/icon/awesome.png) [PyMT](https://github.com/csdms/pymt) – ![Python](media/icon/python.png) Python toolkit for coupling models and datasets that expose the [Basic Model Interface (BMI)](https://github.com/csdms/bmi)
```

Comment:
In Line 40, you could replace the link to bmi by a link to e.g. the bmi documentation, otherwise the link checker stumbles over a duplicate.

Reply:
Will do!

Reply:
Travis complained about the link to the BMI documentation, so I'm just going to strike the link.

Reply:
Instead of removing the link, you could also add `<a name="bmi"></a>` in the line where the BMI link is already mentioned and add a cross-reference: `[BMI](#bmi)`